src/redisearch_rs/c_entrypoint/varint/cbindgen.toml

```
@@ -0,0 +1,12 @@
+language = "C"
+autogen_warning = "/* Warning, this file is autogenerated by cbindgen from `src/redisearch_rs/c_entrypoint/varint/build.rs. Don't modify it manually. */"
+cpp_compat = true
+pragma_once = true
+includes = ["../../buffer.h", "../../redisearch.h"]
```

```suggestion
includes = ["buffer.h", "redisearch.h"]
```
I might be wrong, but it should work with just this.

---

Yeah, they are enough. Changed them in 81bfb48c0